JASP-Engine/JASP/R/correlation.R

```diff
@@ -16,7 +16,7 @@
 #
 
 Correlation <- function(dataset=NULL, options, perform="run", callback=function(...) 0, state=NULL, ...) {
-  
```

Comment:
Can you put all stylistic/markup stuff not related to the fix in a separate commit?
This allows us to more easily spot actual changes in the git history of our files.